index.html

```diff
@@ -3,6 +3,17 @@
 title: "Home"
 excerpt: "Resources to get started with Algorithmia"
 exclude_from_search: true
+redirect_from:
+  - /lang/swift/
+  - /guides/sentiment-analysis/
+  - /application-development/lang-guides/
+  - /application-development/lang-guides/aws-lambda/
+  - /application-development/lang-guides/java/
+  - /application-development/lang-guides/javascript/
+  - /application-development/lang-guides/node/
+  - /application-development/lang-guides/ruby/
+  - /application-development/lang-guides/rust/
+  - /application-development/lang-guides/scala/
```

Comment:
I'd run this by @mgandco, but I'd assume it'd be more appropriate to redirect `/developers/application-development/lang-guides/*` to the corresponding pages on `/developers/clients/*` since it's more helpful than simply redirecting to the home page.